gke-e2e-cluster-create/0.1/gke-e2e-cluster-create.yaml

```diff
@@ -0,0 +1,107 @@
+    tekton.dev/displayName: "GKE End to End Cluster Create"
+spec:
+  description: |
+    Create a GKE cluster for end to end tests.
+
+    This Task can be used to create a GKE cluster in a GCP project and fetch a kubeconfig that
+    can be used (in a context with both kubectl and gcloud available) to make requests to the cluster.
+
+  params:
+  - name: project-name
+    description: The name of the GCP project in which to create the GKE cluster.
+  - name: private-key-path
+    description: The path to the private key within the gcp-service-account workspace.
+  - name: identifier
+    description: A string which identifies the purpose for which this cluster is being created. Used to name other resources created.
+  - name: min-nodes
+    description: The minimum number of nodes in the cluster.
+    default: "1"
+  - name: max-nodes
+    description: The maximum number of nodes in the cluster.
+    default: "3"
+  - name: region
+    description: The region to create the cluster in.
+    default: us-central1
+  - name: machine-type
+    description: The machine type to create, from https://cloud.google.com/compute/docs/machine-types.
+    default: n1-standard-4
+  - name: image-type
+    description: The type of image to create the nodes, from https://cloud.google.com/kubernetes-engine/docs/concepts/node-images.
+    default: cos
+  - name: cluster-version
+    description: |
+      The GKE version to install, in a format that can be used as the `--cluster-version` argument to
+      https://cloud.google.com/sdk/gcloud/reference/beta/container/clusters/create
+    default: latest
+  workspaces:
+  - name: gcp-service-account
+    description: A Secret or volume containing the private key of a GCP service account that can create GKE clusters in the project
+  - name: kubeconfig
+    description: |
+      A workspace into which a kubeconfig file called `kubeconfig` will be written that will contain the information
+      required to access the cluster. The `kubeconfig` will expect to use gcloud to authenticate, so in order for it to
+      be used it must be run in a container which contains both kubectl and gcloud.
+  results:
+  - name: cluster-name
+    description: The name of the cluster that was created.
+  steps:
+  - name: gcloud
+    image: google/cloud-sdk:slim@sha256:27b2c22bf259d9bc1a291e99c63791ba0c27a04d2db0a43241ba0f1f20f4067f
+    script: |
+      UNIQUE_STR=$(head /dev/urandom | tr -dc a-z0-9 | head -c 10 ; echo '')
+      UNIQUE_NAME=$(params.identifier)-$UNIQUE_STR
+
+      # Configure gcloud to use the provided project and service account
+      gcloud auth activate-service-account --key-file=$(workspaces.gcp-service-account.path)/$(params.private-key-path)
+      gcloud config set project $(params.project-name)
+
+      # Create a network and a new cluster
+      gcloud compute networks create $UNIQUE_NAME --project $(params.project-name) --subnet-mode=auto
+      gcloud beta container clusters create \
```

Comment:
Why do we need to use beta here?

Reply:
i don't know! thats what kubetest was doing but ill remove it and if that still works then 👍